examples/example-multiplot.cpp

```diff
@@ -0,0 +1,57 @@
+// sciplot - a modern C++ scientific plotting library powered by gnuplot
+// https://github.com/sciplot/sciplot
+//
+// Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+// Copyright (c) 2020 Allan Leal
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+// sciplot includes
+#include <sciplot/sciplot.hpp>
+using namespace sciplot;
+
+int main(int argc, char **argv)
+{
+    // Create 4 different plots
+    const vec x = linspace(0.0, 5.0, 200);
+    plot p0;
```

Comment:
I think this nice example would be better presented if each block of plot commands are separated by empty lines.

~~~cpp
plot p0;
p0.draw(x, std::sin(x)).title("sin(x)");
p0.draw(x, std::cos(x)).title("cos(x)");

plot p1;
p1.draw(x, std::cos(x)).title("cos(x)");

plot p2;
p2.draw(x, std::tan(x)).title("tan(x)");

plot p3;
p3.draw(x, std::sqrt(x)).title("sqrt(x)");
~~~